### PR TITLE
[RTM] Hotfix/subdcawidget

### PR DESCRIPTION
**Pull request by zonky2**
Hotfix/subdcawidget

* fix error is value null
* fix subdcawidget table

### CODE REVIEW

File: src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php [hunk 1]
```diff
@@ -381,7 +381,7 @@ public function saveCallback(string $value = null)
      */
```
Comment (discordier) [inline src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php]: The `@return string` should now be changed to `@return string|null`

File: src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php [hunk 2]
```diff
@@ -369,7 +369,7 @@ protected function buildFilterParamsFor(\DC_Table $dataContainer, $elementName)
...
```
Comment (discordier) [inline src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php]: The `@return string` should now be changed to `@return string|null`

File: src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php (hunk 1 above)
Comment (zonky2) [reply src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php]: done

Comment (zonky2) [reply src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php]: done

File: src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php (hunk 2 above)
Comment (zonky2) [reply src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php]: done

**Pull request merged**